### DISCUSSION TITLE
UX/DX: Git LFS should be on by default

### DISCUSSION THREAD

**Discussion by elmarsto** [vercel/community#3220] [Idea]
### Page to Investigate

https://vercel.com/changelog/git-lfs-support

### Steps to Reproduce

When importing a project into Vercel that uses Git LFS, it's a submarined gotcha when you need to navigate into a settings panel and enable a specific option in order for your repo type to be correctly supported. 

By default, LFS should be detected and the option automatically enabled.

Better, the LFS option should be removed entirely from the settings panel, and LFS should be transparently/automatically used.

See see https://github.com/orgs/vercel/discussions/3216

**Comment by amyegan** [comment]
Git LFS is not enabled by default because it can cause slower build times for everyone. However, the team plans to add automatic LFS detection and related features to make deployment easier for projects that use it.
